### ISSUE TITLE
Maintainer Needed!

### ISSUE THREAD

**Issue by bringking** [bringking/requirejs-maven-plugin#11]
This plugin is still very useful for projects using Java and Maven, however I no longer use this day to day and it has become something I can no longer maintain. If anyone wants to fork and pick up the torch,  respond here and I will update the readme to point to your fork. Also, if you need any guidance on doing a release to Maven central I can help on that.

**Comment by ChuanyuWang** [comment]
Indeed, this plugin is quite useful. I use it in our project which has around 100 developers working on it. Somehow it has some defect, but it provides the basic function we want. I even submit a bug a couple months ago. Personally, I don't want to see this plugin out of maintenance. I have fork this project under my account. I would like to take over this torch and try to fix the bug sumbitted by myself. I hope I am will still good at Java -___-!, and release to Maven central is next thing I have to catch up.

**Comment by bringking** [comment]
@ChuanyuWang Thank you for taking the torch on this. I will update the Readme to point to your fork.

**Comment by bringking** [comment]
@ChuanyuWang do you want to take ownership of this repo so you can track issues?

**Comment by ChuanyuWang** [comment]
I will visit this repo sometimes, and check if there is any new issue. So you can keep it. :)
I may come back to you later about release to Maven central, or ask for help from bringking@gmail.com?

**Comment by bringking** [comment]
Okay that sounds good. Yeah feel free to email me if you need a hand with publishing to maven central.

**Comment by longtimeago** [comment]
@bringking I'm also can take a part in the project maintenance. It's quite useful!

**Comment by bringking** [comment]
No worries, @longtimeago I am still accepting PRs against this library for people to want to build and have in their local repo, however I am not actively deploying new releases to maven central.

**Comment by bringking** [comment]
Nor am I testing/fixing any of the open issues on the repo, so any help you could provide on those would be much appreciated.
